src/app/pages/fund/fund.layout.tsx

```diff
@@ -40,9 +40,7 @@ export function FundLayout(props: FundLayoutProps) {
           textAlign={['left', 'center']}
         >
           {/* TODO: Investigate removing / refactoring <PageTitle */}
-          <styled.h1 textStyle="heading.02">
-            {desktopViewport ? 'Let’s get STX into your wallet' : 'Get STX'}
-          </styled.h1>
+          <styled.h1 textStyle="heading.02">Let's get STX into your wallet</styled.h1>
```

Comment:
In the latest design we show the full sentence here

![Screenshot 2023-09-12 at 10 43 19](https://github.com/leather-wallet/extension/assets/2938440/555eb68b-dfca-4e04-808e-6f17ab0fdece)